{"_id":"f8c8da03-f469-3dbf-9765-a9c5c1942c4a","authorName":"Funbit","authorEmailEncrypted":"G5jlBgELovwAL5tHwDTKb22CfLlv6CFbcPD6sYJbmSYUFI/GP3IJx6+fNIbss6De3ZnABJ7DISEs3MqkYnaNng5ddGUxYBiDUEzOHLfxRqa2NHT/7cq6tKHL+axccKIvv2B8GkThfGWE5Ryphon+DPl8JytdLCmAf1SANRU6VRRisHk5q2Z6WSeBsjt2XLYkl6eOZPwknhoT7RZ4agP7qLjZUFljTkqfkvEl11wR6dyTVwC76iFplq3Iuzt0+xYYFFq25APGFsvh3cIlj/15+qBABTExSWuMlvKecErboVcexvIcm6fgutK66uN8wTLvrOGM2xLmErO2O0kEJmbYQNKxP98f3Ez9bcpDj6D/9MB+GQSAa7hhdtuDPcAwmYi4GkWhisSTaOP0zBmiZ8iwxv1v6Kk7zDjurufMVATLHMt+xoqzQ4X1H0Fn6GkQXoHxS2P0WZEg9GNf2gnyR2KJldSauAXW+EBk7+v7v7yZM97jfgPg9cwwpE9M2zZHEOR3Q0O4TVv1UbFNmcnxLK7CYFuaNYSF8SHCp1xzSEUTULloAW9MvDV535m5/oSU8WNpS0wvrxCowfYxnVFVqM/WNaEnEGefP0mFsbjWSXgKfh+BlW4lfnoipmxrpXs8kpTRiHbCyuf0SoZS5dSpvUiROSP6Q4t4Hm7V0+KZY3LG9U8=","authorEmailMD5":"1b66f6609067719d158ee9e7187b1f3b","message":"I think Brent Arias was right. On my side that \"...not marked serializeable...\" exception throws when I try to call my managed C++ library from C# code with something left in the logical CallContext. I had to clear CallContext before the call and restore it back when I get result... Haven't found any other solution.","postId":"2013_04_implicit-async-context-asynclocal-f13467e9-235a-3d80-9d73-143a2fd30bb5","replyTo":"744fad73-022f-3c0f-849d-b8319b25a376","date":"2015-10-01T05:28:14Z","timestamp":1443677294,"authorUserId":"disqus:Funbit"}